flutter-idea/src/io/flutter/run/daemon/DevToolsService.java

```diff
@@ -119,12 +123,38 @@ private void startServer() {
       }
 
       if (dartDevToolsSupported) {
+        // This condition means we can use `dart devtools` to start.
         final WorkspaceCache workspaceCache = WorkspaceCache.getInstance(project);
         if (workspaceCache.isBazel()) {
+          // This is only for internal usages.
           setUpWithDart(createCommand(workspaceCache.get().getRoot().getPath(), workspaceCache.get().getDevToolsScript(),
                                       ImmutableList.of("--machine")));
         }
         else {
+          final String localDevToolsDir = Registry.stringValue(LOCAL_DEVTOOLS);
```

Comment:
would it also be possible to specify arguments that should be passed to the `devtools_tool serve` command? For example, VS Code allows a list of "args" to be specified where we can add any additional flags that need to be passed (like `--wasm`) for example.

Reply:
Sure, I think I could add a separate registry field for those